conf/lang.yaml

```diff
@@ -285,6 +285,11 @@ MI_Export_Image:
   rs: Дијаграм у имиџ фајл
   ru: Диаграмму в файл изображения
   de: Diagram als Bilddatei exportieren
+MI_Export_RTF:
```

Comment:
Where is the RTF export support? If it is not part of this PR, these changes should be removed.

Reply:
i removed it, it not finalised.